api/tools/index.ts

```
@@ -1,20 +1,21 @@
+// import { registerBotTools } from "./bots";
 import { registerEchoTool } from "./utils/echo";
+import { registerJoinSpeakingTool } from "./bots/join-speaking";
 
 export function registerTools(server: McpServer, apiKey: string): McpServer {
   const baasClient = new BaasClient({
     apiKey: apiKey,
   });
 
   // Register bot tools
-  const updatedServer = registerBotTools(server, baasClient);
+  let updatedServer = registerJoinSpeakingTool(server);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updatedServer' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)